source/index.html.md

```diff
@@ -630,6 +630,31 @@ object][ApplePay-PaymentToken] for more information.
 </p>
 
 
+##### Method: `samsungpay`
+
+Samsung Pay provides the payment details as either JWE or JWE/JWS JOSE objects. Currenlty we only
+accept the JWE.
+Your system must extract the content encryption key (<code>cek</code>) used to encrypt the JWE
```

Comment:
```suggestion
Your system must extract the content encryption key (CEK) used to encrypt the JWE
```

... but we don't seem to use the CEK acronym anywhere?

Reply:
This is to refer to the JWE spec: https://tools.ietf.org/html/rfc7516#section-2

It should be upper case however.